spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestViews.java

```diff
@@ -562,6 +562,64 @@ public void readFromViewWithCTE() throws NoSuchTableException {
     assertThat(sql("SELECT * FROM %s", viewName)).hasSize(1).containsExactly(row(10, 1L));
   }
 
+  @TestTemplate
+  public void readFromViewWithGroupByOrdinal() throws NoSuchTableException {
+    insertRows(3);
+    insertRows(2);
```

Comment:
not sure why we actually need to first write 3 rows and then 2 rows. I believe this line can be removed and the result check further below can be updated to `.containsExactlyInAnyOrder(row(1, 1L), row(2, 1L), row(3, 1L))`

Reply:
use duplicate value to Verify that the group by command is effective.